src/app/tests/suites/certification/Test_TC_DM_2_2.yaml

```diff
@@ -28,65 +31,92 @@ tests:
               - name: "nodeId"
                 value: nodeId
 
-    - label: "Query NOCs"
+    - label: "Query SoftwareVersion"
       command: "readAttribute"
-      # TODO: Enabled and update value when SDK supports this.
-      disabled: true
-      attribute: "NOCs"
-      PICS: DM_ICACERT
+      attribute: "SoftwareVersion"
       response:
-          value: [{ NOC: 0 }]
+          saveAs: SoftwareVersionValue
           constraints:
-              type: list
-
-    - label: "Query NOCs"
-      command:
-          "readAttribute"
-          # TODO: Enabled and update value when SDK supports this.
-      disabled: true
-      attribute: "NOCs"
-      PICS: DM_ICACERT
-      response:
-          value: [{ NOC: 0, ICAC: 0 }]
-          constraints:
-              type: list
+              type: uint8
 
-    - label: "Query fabrics list"
+    - label: "Query Reachable Fabrics"
       command: "readAttribute"
-      attribute: "Fabrics"
+      attribute: "Reachable"
       response:
-          value: [
-                  {
-                      # We really don't know what values to expect for any of
-                      # the fields, except Label.
-                      Label: "",
-                  },
-              ]
           constraints:
-              type: list
+              type: bool
 
-    - label: "Query Supported Fabrics"
-      command: "readAttribute"
-      attribute: "SupportedFabrics"
-      response:
-          value: 16
-          constraints:
-              type: uint8
+    - label: "Reboot target device"
+      cluster: "SystemCommands"
+      command: "Reboot"
+      arguments:
+          values:
+              - name: "discriminator"
+                value: discriminator
```

Comment:
Reboot does not take those arguments. It takes no arguments at all.
```suggestion
```

Reply:
i will update